cmd/api/main.go

```diff
@@ -77,6 +77,10 @@ func main() {
 			user.POST("/", userHandler.CreateUser)
 			user.POST("/load-by-csv", userHandler.LoadUserByCSV)
 			user.POST("/reset-password", userHandler.ResetPassword)
+			admins := user.Group("/administrators", middleware.AdminMiddleware(authService))
+			{
+				admins.POST("/register-admin", userHandler.CreateUser)
+			}
```

Comment:
We do not need to create a new route to create managers. It is a bad practice to use a handler for more than 1 route in the application. Please use the route on line 77 and delete these lines.